lib/react_client.dart

```diff
@@ -776,60 +935,12 @@ ReactDartComponentFactoryProxy2 _registerComponent2(
   return new ReactDartComponentFactoryProxy2(reactComponentClass);
 }
 
-/// Creates ReactJS [ReactElement] instances for DOM components.
-class ReactDomComponentFactoryProxy extends ReactComponentFactoryProxy {
```

Comment:
This wasn't removed, it was just moved up to group it with other [FactoryProxy]s